src/posthog-core.ts

```diff
@@ -983,9 +989,11 @@ export class PostHog {
         if (!this.config.disable_scroll_properties) {
             let performanceProperties: Record<string, any> = {}
```

Comment:
fly-by since i've never actively read this section... not jazzed about `performanceProperties` since it's not really about performance. could be `pageViewProperties` or something but really very nitty 😅